src/commands/help.ts

```diff
@@ -36,21 +36,23 @@ const command: Command = {
 				.catch(console.error.bind(console));
 		} else {
 			// Display specific command information.
-			const commandObject = getCommand(command);
+			const commandObject = getCommand(commandName);
 
-			if (!commandObject || !commandObject.hasPermission(state, message)) {
-				ephemeral(state, message.reply("The command does not exist.")).catch(console.error.bind(console));
+			if (!commandObject || !commandObject.hasPermission(state, interaction)) {
+				interaction
+					.reply("The command does not exist.", { ephemeral: command.shouldBeEphemeral(state, interaction) })
+					.catch(console.error.bind(console));
 				return;
 			}
 
-			message
+			interaction
 				.reply(
 					new MessageEmbed({
 						fields: [
 							{
 								name: "Command",
 								value: `${commandObject.name} ${commandObject.options
-									.map(option => (option.optional ? `*[${option.name}]*` : `*${option.name}*`))
+									.map(option => (option.required ? `*[${option.name}]*` : `*${option.name}*`))
```

Comment:
Did this change carry over correctly?

I always get the `[ ]` for required mixed up. I'd assume they are still flipped, considering when `option.optional = true` it would have been `[name]`, but now it's `option.required == true`, which is the opposite of the former.

Reply:
Yeah the operands need to be flipped, so the non-required (optional) is in brackets.
```suggestion
									.map(option => (option.required ? `*${option.name}*` : `*[${option.name}]*`))
```